overseer/overseer.py

```diff
@@ -0,0 +1,163 @@
+"""
+Stencila Hub Overseer.
+
+A Celery app for monitoring job and worker events.
+A worker must be run with the `--events` to emit events.
+See http://docs.celeryproject.org/en/stable/userguide/monitoring.html#events.
+"""
+from datetime import datetime
+import os
+import warnings
+
+from celery import Celery
+import httpx
+
+celery = Celery("overseer", broker=os.environ["BROKER_URL"])
+
+# Setup API client
+api = httpx.Client(
+    base_url=os.path.join(os.environ["DIRECTOR_URL"], "api/"),
+    headers={"content-type": "application/json", "accept": "application/json",},
+    timeout=30,
+)
+
+
+def update_job(id, data):
```

Comment:
I'm going to make a swear jar and you have to put in 10c every time you use a variable called `id` or `type`.